src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/default.phpt

```diff
@@ -0,0 +1,62 @@
+--TEST--
+Test DeprecationErrorHandler in default mode
+--SKIPIF--
+<?php if (!file_exists('DeprecationErrorHandler.php')) print "skip"; ?>
+--FILE--
+<?php
+
+putenv('SYMFONY_DEPRECATIONS_HELPER');
+
+define('PHPUNIT_COMPOSER_INSTALL', __DIR__.'/../../vendor/autoload.php');
```

Comment:
btw, would be it possible to support the case of running these tests from the root too ? It would make it easier for contributors to run them.